src/web/components/Core/Dialog.spec.tsx

```diff
@@ -18,13 +18,20 @@ describe('Dialog', () => {
 
   it('does not render text close button if closeButton is undefined', () => {
     render(<WithoutCloseText />);
+    const openButton = screen.getByText('Open Dialog');
+    fireEvent.click(openButton);
+
     expect(screen.queryByText('Close')).not.toBeInTheDocument();
-    expect(screen.queryByLabelText('Close')).not.toBeInTheDocument();
+    expect(screen.getByLabelText('Close')).toBeInTheDocument();
```

Comment:
I think we can replace this with the preferred: `expect(screen.getByRole('button', { name: 'Close' })).toBeInTheDocument();`

Reply:
Ah yeah sounds good